src/newgrf.cpp

```diff
@@ -9188,7 +9190,7 @@ static void FinalisePriceBaseMultipliers()
 		GRFFile *dest = GetFileByGRFID(override);
 		if (dest == NULL) continue;
 
-		grf_overrides[i] = _grf_files.FindIndex(dest);
+		grf_overrides[i] = std::find(_grf_files.begin(), _grf_files.end(), dest) - _grf_files.begin();
 		assert(grf_overrides[i] >= 0);
```

Comment:
this assert is now wrong - should be check for size() (or the construct should use iterators instead of indices)